Extending/Packages/Creating-a-Package/index.md

```diff
@@ -0,0 +1,146 @@
+
+Finally after filling out all the info we can either `Save` or `Download` the package. We will download it, in order to take a closer look at what it contains in the zip.
+
+## Inspecting a package zip
+
+When you download and then open the zip package you will find that it looks like this:
+
+![Content of a zip package](images/zip-package.png)
+
+The 5 highlighted files were the ones the package contained that we created in the [Creating a Custom Dashboard tutorial](../Creating-a-Custom-Dashboard/index.md), however there is another file here called `package.xml` - so let's take a look at that - it looks like this:
+
+```xml
+<?xml version="1.0" encoding="utf-8"?>
+<umbPackage>
+  <files>
+    <file>
+      <guid>customwelcomedashboard.controller.js</guid>
+      <orgPath>~/App_Plugins/CustomWelcomeDashboard</orgPath>
+      <orgName>customwelcomedashboard.controller.js</orgName>
+    </file>
+    <file>
+      <guid>customwelcomedashboard.css</guid>
+      <orgPath>~/App_Plugins/CustomWelcomeDashboard</orgPath>
+      <orgName>customwelcomedashboard.css</orgName>
+    </file>
+    <file>
+      <guid>package.manifest</guid>
+      <orgPath>~/App_Plugins/CustomWelcomeDashboard</orgPath>
+      <orgName>package.manifest</orgName>
+    </file>
+    <file>
+      <guid>WelcomeDashboard.html</guid>
+      <orgPath>~/App_Plugins/CustomWelcomeDashboard</orgPath>
+      <orgName>WelcomeDashboard.html</orgName>
+    </file>
+    <file>
+      <guid>en-US.xml</guid>
+      <orgPath>~/App_Plugins/CustomWelcomeDashboard/lang</orgPath>
+      <orgName>en-US.xml</orgName>
+    </file>
+  </files>
+  <info>
+    <package>
+      <name>Custom Welcome Dashboard</name>
+      <version>1.0.0</version>
+      <iconUrl></iconUrl>
+      <license url="http://opensource.org/licenses/MIT">MIT License</license>
+      <url>https://github.com/umbraco/customwelcomedashboard</url>
+      <requirements type="Strict">
+        <major>8</major>
+        <minor>2</minor>
+        <patch>0</patch>
+      </requirements>
+    </package>
+    <author>
+      <name>Jesper Mayntzhusen</name>
+      <website>https://jesper.com</website>
+    </author>
+    <contributors></contributors>
+    <readme><![CDATA[This will add a dashboard to your content section. 
+
+The dashboard will show each user the most recent nodes they have saved.]]></readme>
+  </info>
+  <DocumentTypes />
+  <Templates />
+  <Stylesheets />
+  <Macros />
+  <DictionaryItems />
+  <Languages />
+  <DataTypes />
+  <Actions />
+</umbPackage>
+```
+
+You will notice that each of the fields we created is inside this XML file. And all files have an `orgPath` attached to it that it will try to move the file to when installing it - so don't worry if you organized your package in folders and they are not so in the zip.
```

Comment:
```suggestion
You will notice that each of the fields we created is inside this XML file. All files have an `orgPath` attached to it - this is where it will try to move the file to when installing it. So, don't worry if you organized your package in folders and they are not so in the zip.
```